frontend/src/scenes/feature-flags/FeatureFlagInsightsCard.tsx

```diff
@@ -0,0 +1,29 @@
+import { useValues } from 'kea'
+import { CompactList } from 'lib/components/CompactList/CompactList'
+import React from 'react'
+import { InsightRow } from 'scenes/project-homepage/RecentInsights'
+import { urls } from 'scenes/urls'
+import { InsightModel } from '~/types'
+import { featureFlagLogic } from './featureFlagLogic'
+
+export function FeatureFlagInsightsCard(): JSX.Element {
```

Comment:
nit: rename to RecentFeatureFlagInsightsCard to be consistent with other Recent* components